champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/axe/Flash.java

```diff
@@ -54,12 +54,12 @@ public String getName() {
     @Override
     public String[] getDescription(int level) {
         return new String[] {
-                "Teleport a short distance horizontally",
-                "in the direction you are facing.",
+                "Right click with an Axe to activate",
                 "",
-                "Uses up to <val>" + getMaxCharges(level) + "</val> charges.",
+                "Teleport 5 Blocks in the direction you are facing",
```

Comment:
I think there is a teleportDistance variable that could replace 5. And also give that the <stat> tag.

Problem was I could never figure out where the 5 distance was used so I did not touch it.

Reply:
you are right ill need to modify this to get the config